index.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>West African Fiction Festival-Home</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+  <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <script defer type="module" src="index.js"></script>
+</head>
+<body class="flex">
+  <main>
+    <header>
+      <div class="pre-nav sub-section-grey">
+        <ul><a href="#"><i class="fa fa-facebook"></i></a></ul>
+        <ul><a href="#"><i class="fa fa-twitter"></i></a></ul>
+        <ul><a class="sub-section-text-white" href="#">English</a></ul>
+        <ul><a class="sub-section-text-white" href="#">My Page</a></ul>
+        <ul><a class="sub-section-text-white" href="#">Logout</a></ul>
+      </div>
+      <nav class="mobile-nav flex">
+        <div id="menu-btn" data-menu>
+          <div class="menu-bar"></div>
+          <div class="menu-bar"></div>
+          <div class="menu-bar"></div>
+        </div>
+        <a href="index.html"><img class="logo-mobile" src="images/logo2-removebg-preview.png" alt=""></a>
+      </nav>
+      <nav class="desktop-nav flex display-none">
+        <a href="#"><img class="logo" src="images/logo2-removebg-preview.png" alt="African Fiction Festival 2023 Logo"></a>
+        <ul class="flex">
+          <li><a href="index2.html">About</a></li>
+          <li><a href="#program-section">Program</a></li>
+          <li><a href="#speakers-section">Join</a></li>
+          <li><a href="#partners">Sponsors</a></li>
+          <li><a href="index2.html/#see-past-header">News</a></li>
```

Comment:
Your design
![image](https://user-images.githubusercontent.com/78251927/233439302-c12ec445-8b60-4a35-a814-e221199d70a4.png)

Template
![image](https://user-images.githubusercontent.com/78251927/233439399-1ac66cee-04a2-49ec-9a68-39fee4c8643d.png)

Please kindly remove the image 👆  as it is not present in the original design. 👀👍

Part of the learning objective of this capstone project is to demonstrate our understanding of how to use medium-fidelity wireframes to create a UI. 🤗🛠


We can include a link to navigate back to the `Home page` in the nav list in the mobile version design to ensure the possibility to navigate between the pages of your website improving the user's experience. 🤗🛠